modin/pandas/concat.py

```diff
@@ -132,3 +145,35 @@ def concat(
         else:
             result_df.columns = new_idx
     return result_df
+
+
+def _determine_name(objs: Iterable[BaseQueryCompiler], axis: Union[int, str]):
+    """
+    Determine names of index after concatenation along passed axis
+
+    Parameters
+    ----------
+    objs : iterable of QueryCompilers
+        objects to concatenate
+
+    axis : int or str
+        the axis to concatenate along
+
+    Returns
+    -------
+        `list` with single element - computed index name, `None` if it could not
+        be determined
+    """
+    axis = pandas.DataFrame()._get_axis_number(axis)
```

Comment:
Shouldn't there be used Modin `_get_axis_number` function as it is done in `modin.pandas.base.py`?

Reply:
If you mean `pd.DataFrame()._get_axis_number()` then I think that using `pandas` for getting axis number will be more performant. Modin frame constructor besides creating empty pandas dataframe will try to distribute it, which takes a decent time in comparison with just getting axis number from empty pandas dataframe